ooni/settings.py

```diff
@@ -156,6 +156,9 @@
 
 # These are the the embedded settings
 _SETTINGS_INI = os.path.join(OONIPROBE_ROOT, 'settings.ini')
+# This can be overridden by an environment variable for end to end testing
+if 'OONIPROBE_SETTINGS' in os.environ:
+    _SETTINGS_INI = os.environ['OONIPROBE_SETTINGS']
```

Comment:
Yes that is true, but I prefer to have it be explicit and clear that this is what is going on here. Moreover we may want to override other settings via environment variables and it's good to have a consistent pattern for doing so.